validation/jvm/src/main/scala/hmda/validation/rules/ts/syntactical/S028.scala

```diff
@@ -0,0 +1,15 @@
+package hmda.validation.rules.ts.syntactical
+
+import hmda.model.fi.ts.TransmittalSheet
+import hmda.validation.dsl.{ HmdaDsl, CommonDsl, Result }
+
+/*
+ Timestamp must be numeric and in ccyymmddhhmm format
+ */
+object S028 extends CommonDsl with HmdaDsl {
+  def apply(ts: TransmittalSheet): Result = {
+    import scala.language.postfixOps
+    val timestamp = ts.timestamp
+    (timestamp is numeric) and (timestamp.toString is validTimestampFormat)
```

Comment:
The division between parser and validation is totally arbitrary and defined by how the edits are prescribed in HMDA. If I were to design this from scratch, I would put a lot more intelligence through the type system into the data model / parser. But one design goal has always been "implement all the rules" and "make them flexible". So I think the parser can't be responsible for S028, unless those initial requirements change